app/controllers/paths_controller.rb

```diff
@@ -46,4 +44,12 @@ def destroy
   def path_params
     params.require(:path).permit(:title, :description)
   end
+
+  def find_path
+    begin
+      @path = Path.find_by_title(params[:id])
+    rescue ActiveRecord::RecordNotFound
+      redirect_to action: :index
+    end
```

Comment:
this block won't raise any error since it will return `nil` if `Path` not found. Only `Path.find()` would raise error